addon/components/sortable-item.js

```diff
@@ -239,11 +243,20 @@ export default class SortableItemComponent extends Component {
   _onDrop() {
     this.args.dragend ? this.args.dragend() : '';
 
+    if (!this.sortableContainer?.cloneNode) {
+      return;
+    }
```

Comment:
onDrop is called on mouseup after a mousedown.
Let's say the use dows a mousedown but does not drag. This means clone node is not created and onDrop need not be called in this case.